src/caffe/test/test_gradient_based_solver.cpp

```diff
@@ -521,7 +531,7 @@ TYPED_TEST(NesterovSolverTest, TestNesterovLeastSquaresUpdateWithMomentum) {
   const Dtype kMomentum = 0.5;
   const int kNumIters = 1;
   for (int i = 0; i <= kNumIters; ++i) {
-    this->TestLeastSquaresUpdate(kLearningRate, kWeightDecay, kMomentum, i);
+    this->TestLeastSquaresUpdate(kLearningRate, kWeightDecay, kMomentum, 0., i);
```

Comment:
These should be declared as constants (e.g. `const Dtype kRMSDecay = 0`) like the other args to make the meaning clear.